command-line/test/Command/Key/HashSpec.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Command.Key.HashSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Test.Hspec
+    ( Spec, SpecWith, it, shouldBe, shouldContain )
+import Test.Utils
+    ( cli, describeCmd )
+
+spec :: Spec
+spec = describeCmd [ "key", "hash" ] $ do
+    specKeyNotPublic
```

Comment:
Could be good to show that it succeeds otherwise 😅

Reply:
added here : https://github.com/input-output-hk/cardano-addresses/pull/76/commits/15d9570fed76ab87e7a00f723f2273965bac1032